src/sdk/vscode/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 ## [Unreleased]
 
+- 0.9.6 [2021-06-07]
```

Comment:
I think that has to be merged with the other PR of you

Reply:
Now, the conflicts are there ;-)

Reply:
Conflict resolved :) Was a cross correlation